packages/pcd/zk-eddsa-event-ticket-pcd/test/ZKEdDSAEventTicketPCD.spec.ts

```diff
@@ -602,10 +602,10 @@ describe("ZKEdDSAEventTicketPCD should work", function () {
       claim.watermark = "111";
     });
     await testVerifyBadClaim(validPCD, (claim: ZKEdDSAEventTicketPCDClaim) => {
-      claim.signer[0] = "123";
+      claim.signer = ["123", "456"];
     });
     await testVerifyBadClaim(validPCD, (claim: ZKEdDSAEventTicketPCDClaim) => {
-      claim.signer[1] = "123";
+      claim.signer = "123456";
```

Comment:
As elsewhere, I think this should be a well-formed signing key, or else it's not really testing the same thing.